server/src/test/java/com/objectcomputing/checkins/services/skills/SkillControllerTest.java

```diff
@@ -0,0 +1,178 @@
+package com.objectcomputing.checkins.services.skills;
+
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.client.HttpClient;
+import io.micronaut.http.client.annotation.Client;
+import io.micronaut.http.client.exceptions.HttpClientResponseException;
+import io.micronaut.test.annotation.MicronautTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import javax.inject.Inject;
+import java.sql.Date;
+import java.util.*;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.*;
+
+@MicronautTest
+public class SkillControllerTest {
+
+    @Inject
+    @Client("/skill")
+    private HttpClient client;
+
+    SkillRepository mockSkillRepository = mock(SkillRepository.class);
+    Skill mockSkill = mock(Skill.class);
+
+    private static Date testDate = new Date(System.currentTimeMillis());
+    private static String testSkillName = "testName";
+    private static boolean pending = false;
+    private static boolean isDataSetupForGetTest = false;
```

Comment:
It looks like isDataSetupForGetTest and testDate aren't being used in the test class. Could we remove these?